recipes/unbaffeld/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "unbaffeld" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/c4/a3/3dc6f9912137aab7da98640f90ac7ab314128c5c417d4f1b6321196ca0e5/UNBAFFELD-0.2.2.tar.gz"
```

Comment:
It's probably better to use the canonical url that we have in the example:

```
url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Reply:
I had trouble with the link but realized it was a capitalization issue with the example one above. Have modified it accordingly.